pkg/rpc/net.go

```diff
@@ -0,0 +1,30 @@
+package rpc
+
+import (
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+type NetApi struct {
+	rpcClient rpcClient
+}
+
+func NewNetApi(ip string, port string) (*AdminApi, error) {
```

Comment:
just a copy-paste mistake `AdminApi` -> `NetApi` (and L17)

Reply:
good catch! thanks

Reply:
fixed!